monitors/rootmonitor/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+find_package(ROOT QUIET)
```

Comment:
This small addition allows the `EUDAQ_BUILD_ONLINE_ROOT_MONITOR` option to appear in the `ccmake` configuration stage (ensures the `ROOT_FOUND` variable presence).